ledfx/api/find_devices.py

```
@@ -1,30 +1,268 @@
+                    remaining = timeout - elapsed
+
+                    if remaining <= 0:
+                        break
+
+                    try:
+                        data, addr = await transport.receive(
+                            timeout=min(remaining, 1.0)
+                        )
+                        response_count += 1
+
+                        header, payload = parse_message(data)
```

Ruff (0.14.14)

[warning] Unpacked variable `payload` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)